File: IssueTracker_210042122/src/IssueRepository.java
import java.util.ArrayList;
import java.util.List;

public class IssueRepository {
    public List<Issue> issues = new ArrayList<Issue>();

public IssueRepository (List<Issue>issues){
    this.issues=issues;
}

    public void CreateIssue(Issue issue) {
            issues.add(issue);
            System.out.println("Success");

    }

    public List<Issue> GetAllIssues() {
        return new ArrayList<>(issues);
    }
}

File: TaskManager/TrelloApp/TrelloApp/IssueRepository.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TrelloApp
{
    public class IssueRepository
    {
        public Issue issue;
        public Dictionary<int, Issue> issues = new Dictionary<int, Issue>();

        public IssueRepository(Issue issue) 
        {
            this.issue = issue;
        }

        public void createIssue (Issue issue)
        {
            issues.Add(issue.id, issue);
        }

        public Issue getIssue (int issueid)
        {
            return issues[issueid];
        }

        public void updateIssue(Issue issue)
        {
            if (issues.ContainsKey(issue.id))
            {
                issues[issue.id] = issue;
            }
        }
    }
}
